libcxx/src/random.cpp

```diff
@@ -41,7 +43,24 @@
 
 _LIBCPP_BEGIN_NAMESPACE_STD
 
-#if defined(_LIBCPP_USING_GETENTROPY)
+#if defined(NANOMIPS_RANDOM)
```

Comment:
Why not just use `defined(__nanomips__)` ?

Reply:
Yes, definitely, thanks! I will fix this.